core/src/test/java/feign/PropertyPojo.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2012-2018 The Feign Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package feign;
+
+public class PropertyPojo {
+
+  private String name;
+
+  public static class ChildPojoClass extends PropertyPojo {
+    private Integer number;
+
+    private String address;
+
+    public Integer getNumber() {
+      return number;
+    }
+
+    public void setNumber(Integer number) {
+      this.number = number;
+    }
+
+    public void setAddress(String address) {
+      this.address = address;
+    }
+
+    private String getAddress() {
```

Comment:
Oh I see, this (`private`) is why address doesn't show up... I'd rename `address` to `internalProperty` or `privateGetterProperty` or something to show (in the test case) why this isn't expected to show up.